recipe/meta.yaml

```
@@ -1,29 +1,31 @@
-{% set version = "1.2.4" %}
+{% set version = "1.2.5" %}
 
 package:
   name: pytoolconfig
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytoolconfig/pytoolconfig-{{ version }}.tar.gz
-  sha256: 5e1a246f77970ddb5f3d8d06fbf162424b8a1adfc22c2eb51826b383bf293d1e
+  url: https://github.com/bagel897/pytoolconfig/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 2d5ee0ed520662f3a895d2890c3e25e2250e58cdf0d2ca6d9e38214a6da22bf1
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
-  number: 1
+  skip: True # [py<37]
+  number: 0
 
 requirements:
   host:
-    - pip <22
+    - pip
     - pdm
     - pdm-pep517 >=1.0.5
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
     - packaging >=22.0
-    - python >=3.7
-    - tomli >=2.0.1
-    - typing-extensions >=4.4.0
+    - python
+    - tomli >=1.2.3  # [py<311]
```

```suggestion
    - tomli >=2.0.1 # [py<311]
```
I think tomli should stay `2.0.1` see: https://github.com/bagel897/pytoolconfig/blob/v1.2.5/pyproject.toml#L7

---

Thank you for spotting that